resources/model/OpenStudio.idd

```diff
@@ -15103,7 +15103,11 @@ OS:Coil:Cooling:DX:SingleSpeed,
        \type object-list
        \required-field
        \object-list QuadraticCubicCurves
-  N6, \field Nominal Time for Condensate Removal to Begin
+  N6 , \field Minimum Outdoor Dry-Bulb Temperature for Compressor Operation
+       \type real
+       \default -25.0
+       \units C
```

Comment:
`OS:Coil:Cooling:DX:SingleSpeed`. 

**The exact same (real, default -25C, units C) is used for all 5 coils.**

**This could be changed to `\required-field`, the default moved to the Ctor (`this->setMinimumOutdoorDryBulbTemperatureforCompressorOperation(-25)`), and remove the `isMinimumOutdoorDryBulbTemperatureforCompressorOperationDefaulted()` and `resetMinimumOutdoorDryBulbTemperatureforCompressorOperation()`**

@kbenne please advise.

Reply:
Yes please. I prefer required fields and initialize in the ctor.